db/schema.rb

```diff
@@ -128,7 +125,6 @@
     t.string "residential_zip"
     t.boolean "self_employed"
     t.integer "self_employed_monthly_income"
-    t.integer "self_employment_expenses"
```

Comment:
hmmmm @hartsick it seems like this was removed accidentally -- rebase and re-migrate?

Reply:
Good catch!